src/wolfgame.py

```diff
@@ -3189,56 +3187,86 @@ def reaper(cli, gameid):
                         var.LAST_SAID_TIME[nick] += timedelta(seconds=10*num_night_iters)
                     num_night_iters = 0
 
-
-            if not skip and (var.WARN_IDLE_TIME or var.PM_WARN_IDLE_TIME or var.KILL_IDLE_TIME):  # only if enabled
+            if not skip and idle_timeouts_enabled:  # only if enabled
```

Comment:
kill the comment, it doesn't explain anything you couldn't already determine from the code itself (I know you didn't add the comment, but you're changing the line anyway :P)